setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='django-airbrake',
-    version='0.0.3',
+    version='0.0.4',
```

Comment:
I have no idea if this is important but there is also a reference to version here: https://github.com/Captricity/airbrake-django/blob/master/django_airbrake/utils/client.py#L65

Reply:
Done